core/commands/storage.go

```diff
@@ -1528,3 +1530,165 @@ the challenge request back to the caller.`,
 	},
 	Type: StorageChallengeRes{},
 }
+
+// Storage Stats
+//
+// Includes sub-commands: info, sync
+var storageStatsCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Get node storage stat in the network.",
+		ShortDescription: `
+btfs storage stat sync <>
+btfs storage stat info <>`,
```

Comment:
no need for this, sub-commands format are auto-populated.

Reply:
ok, but fyi, the other commands have these coded in